Module update: CSS positioning (#40894)
mdn/content: files/en-us/web/css/css_positioned_layout/index.md

sidebar: cssref
---

> The **CSS positioned layout** module defines coordinate-based positioning and offsetting schemes of CSS and the properties used to position and stack elements on a web page. The module is best known for defining the basic positioning methods, including relative positioning, sticky positioning, absolute positioning, and fixed positioning. It also defines how positioned elements are painted and layered, which helps clarify stacking behavior and visual order.

Reviewer: ```suggestion
The **CSS positioned layout** module defines the coordinate-based positioning and offsetting schemes available in CSS and the properties used to position and stack elements on a web page. The module is best known for defining the basic positioning methods, including relative positioning, sticky positioning, absolute positioning, and fixed positioning. It also defines how positioned elements are painted and layered, clarifying stacking behavior and visual order.
```

> The positioned layout module defines the various {{glossary("inset properties")}}. These are logical properties that enable developing with internationalization and responsiveness in mind.

Reviewer: There are physical inset properties too. Are you saying that the positioned layout module is the place where the logical equivalents are specified? I'd clarify this stuff, and also say where the physical ones are specified.

> The positioned layout module defines the various {{glossary("inset properties")}}. These are logical properties that enable developing with internationalization and responsiveness in mind.

Author: ```suggestion
The positioned layout module defines the physical, logical, and shorthand {{glossary("inset properties")}}. The logical properties enable developing with internationalization and responsiveness in mind.
```

> Like all CSS modules, this module impacts and is impacted by other modules. This module describing how positioning interacts with other layout modules, such as [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) and [CSS Grid layout](/en-US/docs/Web/CSS/CSS_grid_layout), while other modules, such as [CSS anchor positioning](), build upon this module to enable positioning elements relative to other elements and layers.

Reviewer: ```suggestion
Like all CSS modules, this module impacts and is impacted by other modules. This module describes how positioning interacts with other layout modules such as [CSS flexible box layout](/en-US/docs/Web/CSS/CSS_flexible_box_layout) and [CSS Grid layout](/en-US/docs/Web/CSS/CSS_grid_layout). Other modules, such as [CSS anchor positioning](), build upon this module to enable positioning elements relative to other elements and layers.
```

Reviewer: You've also missed out a link URL.
